test_recipe.py

```diff
@@ -0,0 +1,44 @@
+import sys
```

Comment:
This file should be moved elsewhere than at the root of the project, ideally to `tests/dogtail` or `tests/broken`

Reply:
As I mentioned in my earlier comment, this file really isn't a "test" and simply a script to launch the UI on its own so it can be manually tested.